02-technical-guides/hierarchies/02-hierarchy-manipulations.ipynb

```diff
@@ -0,0 +1,2029 @@
+{
```

Comment:
<p>Add in a bit at the top to quickly explain what a parent hierarchy is.</p><p>Otherwise I might be a bit lost, and not understand why this is interesting for me.</p><p> </p><p>A quick suggestion:</p><p>A hierarchy is by definition a set of parent-child relationships, and in Atoti we say that a hierarchy is the parent of another when the second hierarchy is a lower level.</p><p>In our hierarchies Products is the parent to Product Category, and so Product Category is the child of Products.</p><p>It is useful to know how hierarchies are structured, as this will help us decide which level of a hierarchy we should analyze, and how far up or down the hierarchy we should aggregate the data to get meaningful insights.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/atoti/pull/890/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>This explanation is good, but I think it is missing a bit about dimensions. If incorporating this bit, let's make sure the relationship between dimensions and hierarchies are clear (and it might require taking a step back to explain what is a dimension).</p>

Reply:
<p>I'm not sure we need to discuss parent/multi-level hierarchies in this passage, as I'm just talking about hierarchies and the dimensions that they fall under. Therefore, I've amended this to:</p><p> </p><p>"The&nbsp;<code>dimension</code>&nbsp;property of each hierarchy returns the name of its parent dimension. This is helpful for when we want to know which dimension a particular hierarchy falls within. For example, we can call the&nbsp;<code>dimension</code>&nbsp;property of the&nbsp;<code>Address</code>&nbsp;hierarchy to determine that its parent dimension is&nbsp;<code>Customers</code>.</p><blockquote>💡&nbsp;<strong>Note:</strong>&nbsp;Remember that when we loaded our tables into our cube, the tables effectively became our dimensions and the non-numeric columns became our hierarchies. Therefore, the&nbsp;<code>Address</code>&nbsp;column of our&nbsp;<code>Customer</code>&nbsp;table became the&nbsp;<code>Address</code>&nbsp;hierarchy under the&nbsp;<code>Customer</code>&nbsp;dimension of our cube, respectively. See the previous cube data model/schema visualization for reference."</blockquote><p> </p>